lib/vault/encrypted_model.rb

```diff
@@ -216,6 +216,8 @@ def parse_transit_attributes(attribute, options)
       # before theirs, resulting in attributes that are not persisted.
       after_save :__vault_persist_attributes!
 
+      before_destroy :__vault_load_attributes!, unless: -> { @__vault_loaded }
```

Comment:
Like the other callbacks above, mind adding a comment here saying why we load attributes before destroying?

Reply:
Coulda sworn I added that comment before pushing. Thanks!